Assignment1/main.css

```diff
@@ -0,0 +1,120 @@
+body{
+    margin: 0;
+    width: 100%;
+    font-family: 'Roboto',sans-serif;
+}
+.headContent{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 1%;
+    margin-right: 1%;
+}
+.mainDivision{
+    margin-left: 1%;
+    margin-right: 1%;
+}
+.descriptionSection{
+    display: flex;
+    justify-content: space-between;
+}
+header{
+    background-color: #1B2354;
+    padding:0.6% 0;
+}
+
+.contentDiv{
+    width: 100%;
+    max-width: 200px;
+    text-align: center;
+    background-color: white;
+    border-style: solid;
+    border-width: 2px;
+    border-color: #ff4848;
+    color:  #1B2354;
+    margin: auto;
+    padding: 0.5%;
+}
+
+.mainTitle{
+    font-size: 1.8rem;
+}
+
+.subTitle{
+    font-size: 0.7rem;
+}
+div h1{
+    margin-top: 0%;
+    margin-bottom: 0%;
+}
+footer{
+    text-align: center;
+    padding:0.5%;
+    text-align: center;
+    color: white;
+    background-color: #273852;
+}
+.subscribeButton{
+    background-color: black;
+    color: white;
+    border-style: none;
+}
+.newsHeading{
+    color:  #273852;
+}
+.descriptionSection{
+    display: flex;
+    flex-direction: row;
+}
+.newsImg{
+    flex-grow:1;
+    align-self: center;
+    padding: 2%;
+    width: 10rem;
+    height: 10rem;
+}
+.newsDesc{
+    margin-right: 12%; 
+    margin-left: 3%;  
+}
+.newsTitle{
+    color:  #273852;
+    padding-bottom: 1%;
+    font-size: 1.8rem;
+}
+.newsDate{
+    font-size:0.7rem;
+}
+.newsDetail{
+    font-size:0.9rem ;

```

Comment:
Here there rem & at the bottom I see px. Pls check

Reply:
ok